swarm/api/manifest.go

```diff
@@ -436,6 +436,16 @@ func (self *manifestTrie) findPrefixOf(path string, quitC chan bool) (entry *man
 	if len(path) <= epl {
 		if entry.Path[:len(path)] == path {
 			if entry.ContentType == ManifestType {
+				err := self.loadSubTrie(entry, quitC)
+				if err == nil && entry.subtrie != nil {
+					subentries := entry.subtrie.entries
```

Comment:
`loadSubTrie()` returns no error, but probably it should? The underlying errors that would bubble up would be from `readManifest()` Have it been evaluated whether they are relevant for this case?

Reply:
? 
`func (self *manifestTrie) loadSubTrie(entry *manifestTrieEntry, quitC chan bool) (err error) {`
looks like it does return an error to me?

Reply:
Sorry, I meant the error is always nil

Reply:
Still not sure I understand:
`
361 func (self *manifestTrie) loadSubTrie(entry *manifestTrieEntry, quitC chan bool) (err error) {
362   if entry.subtrie == nil {
363     hash := common.Hex2Bytes(entry.Hash)
364     entry.subtrie, err = loadManifest(self.dpa, hash, quitC)
365     entry.Hash = "" // might not match, should be recalculated
366   }
367   return
368 }
`
if `loadManifest` returns an error, it would be assigned to `err` on line 364?

Reply:
Still not sure I understand:
`
361 func (self *manifestTrie) loadSubTrie(entry *manifestTrieEntry, quitC chan bool) (err error) {
362   if entry.subtrie == nil {
363     hash := common.Hex2Bytes(entry.Hash)
364     entry.subtrie, err = loadManifest(self.dpa, hash, quitC)
365     entry.Hash = "" // might not match, should be recalculated
366   }
367   return
368 }
`
if `loadManifest` returns an error, it would be assigned to `err` on line 364?

Reply:
Jeez, forget I said anything. I'm a moron.